perf/benchmark/runner/runner.py

```diff
@@ -309,6 +311,15 @@ def generate_nighthawk_cmd(self, cpus, conn, qps, jitter_uniform, duration, labe
 
         return nighthawk_cmd
 
+    def generate_policies(self, security_cmd):
+        os.system("go build {path}generate_policies.go {path}generate.go".format(path=LOCAL_GENERATE_POLICIES))
+        os.system("./generate_policies {param} > {path}policies/{filename}"
```

Comment:
Since policies/ is a folder I created to contain the generated policies I put the policies/ there. I can update LOCAL_GENERATED_SECURITY_POLICY_FILENAME to be policies/generated_policy.yaml but that would not necessarily   be just the filename. What do you think would be best?

Reply:
well, the LOCAL_GENERATED_SECURITY_POLICY_FILENAME is clear that is should just include the filename and not directory, can this be part of the LOCAL_GENERATED_SECURITY_POLICY_DIR? The general rule here is if you are already defining these constants, then all of these path segments should be in these constants instead of repeating them again in the raw path.

Reply:
Sounds good, it would make sense to not have a sub folder and just generate the .yaml file in the generate_policies folder

Reply:
Removed the sub folder